### ISSUE TITLE
There are two "<Label htmlFor" in Toggle component

### ISSUE THREAD

**Issue by chihch** [microsoft/fluentui#12708]
### Environment Information

- **Package version(s)**: 7.94.0
- **Browser and OS versions**: Windows 10

### Describe the issue:

Using Toggle component and open developer tool to inspect component. Notice that there are two "Lable for="Toggle1" in the Toggle component. It’s incorrect from HTML perspective and accessibility. 

Here is a link for reference, https://www.w3.org/TR/html4/interact/forms.html#edef-LABEL.

"The LABEL element may be used to attach information to controls. Each LABEL element is associated with exactly one form control." Is what stated in the link.

Some browsers in fact override label when multiple LABEL FOR is added. So some screenreaders in a different browser might just read On/Off instead of first label.

### Please provide a reproduction of the issue in a codepen:
https://codepen.io/chihch/pen/bGVERpw

#### Actual behavior:

html renders two "Label for="Toggle1"" within the same Toggle component.

#### Expected behavior:

Only one "Label for" associate with the same toggle can be rendered

**Comment by joschect** [comment]
@chihch Thanks for the submission! I can see exactly what you're talking about. I'm guessing this was done to ensure that both labels are read out, but I can see how that could be causing issues. Likely one of those should be an aria-describedby or something like that. We'll get it added to the backlog and fix it.

**Comment by chihch** [comment]
Hi @joschect ,

Thanks for your quick reply. I think you are right.
Quote on my manager who found this bug. The bug here is basically having two LABEL FOR that points to the same control, that should not be happening. Some browsers in fact override label when multiple LABEL FOR is added. So some screen readers in a different browser might just read On/Off instead of first label. He thinks it's a simple fix, just have to remove the second Label For.

**Comment by jdhuntington** [comment]
To add another perspective, the spec says
> Each LABEL element is associated with exactly one form control

but does not say

> Each INPUT element is associated with exactly one label.

I believe this is intentional In fact, [MDN even calls out](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label) that 

> One input can be associated with multiple labels.

Is this causing an issue in your application? Looking at the implementation of `Toggle` there are several things we might be able to do, but it would be useful to know how this is affecting your users.

**Comment by nirbaligar** [comment]
This design causes screen reader issues. Not all browsers concatenate multiple LABEL FOR (pointing to the same target control), so we are noticing scenarios where screen readers just read "Yes/No" value of the switch control without reading the other label, because browsers override one of them. This is a serious issue, because valid label is one of the most important information for screen reader users. Please use aria-describedby to point to the second label "Yes/No".

**Comment by chihch** [comment]
friendly pinging @jdhuntington / @joschect joschect thanks!

**Comment by paulgildea** [comment]
Sounds like this is in PR. Removing needs attention.
